dutch/bos/confrontatie.md

```diff
@@ -0,0 +1,3 @@
+Daarsta je dan, met je zwemnoodle.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.